lib/matplotlib/rcsetup.py

```diff
@@ -446,6 +446,20 @@ def validate_font_properties(s):
     return s
 
 
+def validate_mathtext_fallback(s):
+    fallback_fonts = ['cm', 'stix']
+    if s is None:
+        return s
+
+    if isinstance(s, str):
```

Comment:
I changed this. Not sure, I understand this completely, since [`validate_fontsize`](https://github.com/matplotlib/matplotlib/blob/edd053d16fce81dc604e4e2b9828e04a71155b3f/lib/matplotlib/rcsetup.py#L427) has a similiar check/conversion. But maybe I am missing a subtle point here.

```python
def validate_fontsize(s):
    fontsizes = ['xx-small', 'x-small', 'small', 'medium', 'large',
                 'x-large', 'xx-large', 'smaller', 'larger']
    if isinstance(s, str):
        s = s.lower()
    if s in fontsizes:
        return s
    try:
        return float(s)
    except ValueError:
        raise ValueError("%s is not a valid font size. Valid font sizes "
                         "are %s." % (s, ", ".join(fontsizes)))
```

Reply:
`validate_fontsizes` does also accept numbers.

Reply:
Ok, I see. But I might want to go with your initial suggestion then. Currently, the following code

```python
mpl.rcParams['mathtext.fallback'] = 1
```

leads to the error

```
AttributeError: 'int' object has no attribute 'lower'
```

Of course the user should only supply a string, but the error message might not be very helpful in finding a potential bug in your own code accessing `mpl.rcParams['mathtext.fallback']`.

I added the following

```python
if not isinstance(s, str):
  raise ValueError("Must be a string or None.")
```

which yields the following error message

```
ValueError: Key mathtext.fallback: Must be a string or None.
```

I hope that's ok.